src/meta/mod.rs

```diff
@@ -276,17 +288,57 @@ pub struct Meta {
 }
 
 impl Meta {
+    /// Deserializes `meta`, which contains PGXN `version` metadata, into a
+    /// [`Meta`].
     fn from_version(version: u8, meta: Value) -> Result<Self, Box<dyn Error>> {
         match version {
             1 => v1::from_value(meta),
             2 => v2::from_value(meta),
             _ => Err(Box::from(format!("Unknown meta version {version}"))),
         }
     }
+
+    /// Returns the license string.
+    pub fn license(&self) -> &str {
+        self.license.as_str()
+    }
 }
 
 impl TryFrom<Value> for Meta {
     type Error = Box<dyn Error>;
+    /// Converts the PGXN `META.json` data from `meta` into a [`Meta`].
+    /// Returns an error if `meta` is invalid.
+    ///
+    /// # Example
+    ///
+    /// ``` rust
```

Comment:
Is the space in between intentional?
```suggestion
    /// ```rust
```

Reply:
Yup, reads nicer to me.